<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewGroupDialog</class>
 <widget class="QDialog" name="NewGroupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="NotificationFrame" name="notificationFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="GroupEditor" name="groupEditor">
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="createButton">
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GroupEditor</class>
   <extends>QTabWidget</extends>
   <header>ui/partial/groupeditor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NotificationFrame</class>
   <extends>QFrame</extends>
   <header>ui/partial/notificationframe.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>createButton</sender>
   <signal>clicked()</signal>
   <receiver>NewGroupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>520</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>515</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>NewGroupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>454</x>
     <y>513</y>
    </hint>
    <hint type="destinationlabel">
     <x>516</x>
     <y>501</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
